src/electron/platform/android/setup/live-android-setup-deps.ts

```diff
@@ -31,7 +25,7 @@ export class LiveAndroidSetupDeps implements AndroidSetupDeps {
     public hasAdbPath = async (): Promise<boolean> => {
         try {
             const adbLocation = this.configStore.getState().adbLocation;
-            this.serviceConfig = await this.configFactory.getServiceConfigurator(adbLocation);
+            this.businessLogic = await this.businessLogicFactory.getBusinessLogic(adbLocation);
```

Comment:
This is pretty confusing to read; I don't think it's intuitive that something named `getBusinessLogic()` would throw based on whether ADB is findable or not.